src/components/common/menucard.tsx

```diff
@@ -12,13 +12,15 @@ type Props = {
 const MenuCard: FC<Props> = ({ menu, selectedMenu, onClick }) => {
   return (
     <Card
-      className={`menu-card grid items-center bg-gray-100 rounded-xl h-28 w-36 m-2 cursor-pointer text-center ${
-        selectedMenu === menu.id ? "font-semibold border-2" : "font-normal"
+      className={`menu-card grid items-center rounded-xl h-[90px] w-[158px] m-2 cursor-pointer text-center border-solid border-[#30343B] border-2	${
```

Comment:
remove extra spaces after `border-2`

Reply:
Updated

Reply:
I still see the spaces

Reply:
There are no extra spaces while checking in vs code. The indentation comes with prettier. So it comes in the next line. Otherwise spacings are all fine in the code